.github/workflows/codeql-analysis.yml

```
@@ -121,7 +121,8 @@ jobs:
       run: |
         # Install from vcpkg everything we can for cross-platform consistency
         # If not available, use other methods
-        sudo apt-get install libsecret-1-dev ccache pkg-config pcregrep luarocks expect libzip-dev libglu1-mesa-dev libpulse-dev g++-${{matrix.gcc_compiler_version}} -y
+        sudo apt-get install libsecret-1-dev ccache pkg-config pcregrep luarocks expect libzip-dev libglu1-mesa-dev libpulse-dev g++-${{matrix.gcc_compiler_version}} \
+          libassimp-dev -y
```

For lists like this would it be worthwhile making changes more obvious by insisting on a (sorted?) one item per line type listing?
```suggestion
        sudo apt-get install \
        ccache \
        expect \
        g++-${{matrix.gcc_compiler_version}} \
        libassimp-dev \
        libglu1-mesa-dev \
        libpulse-dev \
        libsecret-1-dev \
        libzip-dev \
        luarocks \
        pcregrep \
        pkg-config \
        -y
```